src/main/java/module-info.java

```diff
@@ -16,7 +16,8 @@
     requires static java.naming;
     requires java.logging;
     requires java.sql;
-
+    requires java.desktop;
```

Comment:
We are adding back java.desktop. This was removed during #144. Can you make a dependency to java.desktop optional, [similar to this factory](https://github.com/eclipse-ee4j/yasson/blob/master/src/main/java/org/eclipse/yasson/internal/components/JsonbComponentInstanceCreatorFactory.java)?

Reply:
Good idea. I'll try that out.